### ISSUE TITLE
Can't close Even Distribution settings.

### ISSUE THREAD

**Issue by Synthlight** [321freddy/even-distribution#21]
Using v1.0.0 of this and Factorio 1.1.

So I click on this to open the settings:
![image](https://user-images.githubusercontent.com/1075166/100170253-39d0fd80-2e93-11eb-85e4-2fb181d72afa.png)

And then I try this to close it:
![image](https://user-images.githubusercontent.com/1075166/100170289-4d7c6400-2e93-11eb-9351-ea8d8929d5af.png)

But this is the result:
![image](https://user-images.githubusercontent.com/1075166/100170333-6d138c80-2e93-11eb-866f-e3d6a22645c4.png)

Basically it doesn't close the settings but removes the top header bar thing.
I tried this on a save from 1.0 and also just started a new game to test this and I get the same results.
I am using the flat character gui if it makes a difference.

**Comment by 321freddy** [comment]
I can't reproduce this. What mods are you using?

**Comment by Synthlight** [comment]
Hmm. I didn't think of that.

Ok, so, after disabling mods 1 by 1, I've narrowed it down to: https://mods.factorio.com/mod/Enhanced_Map_Colors

This + that mod enabled and the above happens. Turn EMC off and the problem goes away.

(I edited EMC's info.json so it loads in 1.1. It seems to work just fine though so I dunno what the incompatibility is.)

**Comment by 321freddy** [comment]
Fixed in version 1.0.1. You were correct, thanks for testing it with other mods :)
